mitodl/ocw-hugo-themes
# Added standalone 404 page

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #608 

#### What's this PR do?
Adds standalone 404 page

#### How should this be manually tested?
- Start ocw-www
- Visit /404.html and verify page is as per requirements 

#### Any background context you want to provide?
FYI
We will need to configure netlify redirects to point towards 404 page
as explained [here](https://gohugo.io/templates/404/)

#### Screenshots (if appropriate)
<img width="989" alt="Screenshot 2022-04-06 at 3 10 19 PM" src="https://user-images.githubusercontent.com/87968618/161952688-3a754f20-bced-46db-9a9b-0cf06ca12f21.png">
<img width="989" alt="Screenshot 2022-04-06 at 3 10 12 PM" src="https://user-images.githubusercontent.com/87968618/161952719-13b8e25e-48a5-4c41-bc1c-6c43fc3260f2.png">
<img width="989" alt="Screenshot 2022-04-06 at 3 10 00 PM" src="https://user-images.githubusercontent.com/87968618/161952730-a91c8e43-7048-4b7b-b174-78a83e4d572d.png">
